homework-submission/react-hyf/react-hyf-client/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "react-hyf-client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-scripts": "3.0.1"
+  },
+  "scripts": {
+    "client-install": "cd .. && cd react-hyf-client && npm install",
```

Comment:
you are going back a directory and going into it again 🤔

Reply:
Yes, you're right... It's my fault, I wanted to put this script in the other package.json of the server folder, so that no need then to navigate to the client folder to install the dependancies. :) But it seems that I've put it in the other one... Corrected!